spaceship/spec/portal/provisioning_profile_template_spec.rb

```diff
@@ -1,4 +1,5 @@
 describe Spaceship::Portal::ProvisioningProfileTemplate do
+  before { Spaceship.login }
```

Comment:
Huh, this is unexpected, but I trust you 😅

Reply:
<img width="1269" alt="image" src="https://github.com/user-attachments/assets/0564afa4-d2af-46f3-95c1-d35911d68d04">